test/inductor/test_torchinductor.py

```diff
@@ -1096,6 +1096,13 @@ def fn(a):
         self.common(fn, ((torch.rand((10, 3, 352, 352), dtype=torch.float32),)))
         self.common(fn, ((torch.rand((14923), dtype=torch.float32),)))
 
+    def test_multilayer_var_lowp(self):
+        def fn(a):
+            return torch.var(a)
+
+        self.common(fn, ((torch.rand((10, 3, 352, 352), dtype=torch.float16),)))
```

Comment:
Does this test repro the issue? Or should we force different number of multi-processsor to simulate the GPU that the author encounter the failure?

Reply:
I've bumped the input size and confirmed it now fails on an A100 machine.